### ISSUE TITLE
Quest Givers: Shindrell Swiftfire & Talen

### ISSUE THREAD

**Issue by zwei7** [HellscreamWoW/Tracker#3286]
These 2 NPCs are offering all their quests all at once instead of one at a time.
![wowscrnshot_102815_162828](https://cloud.githubusercontent.com/assets/15259298/10802113/450b6b6c-7d91-11e5-9dcc-b2697eddf8b7.jpg)
 I am beginning to think this is intentional. That way all the players can play all the quests to see which ones have bugs. This way players won't get stuck on a single quest that may lock out further quests with bugs from being discovered?

**Comment by Guybrush-Hellscream** [comment]
That's correct, if we fixed every quest requirements first, players would get stuck and couldn't progress any further. Also as you said, it's a great way to see which quests are not working and making us gain time to fix them.
